tincr/cad/design/lib_cells.tcl

```diff
@@ -154,7 +155,7 @@ proc ::tincr::get_supported_libcells { } {
 #   as supported if, when instantiated, the reference name of the cell instance matches the name
```

Comment:
I agree. I was planning on doing a pull request that updates a lot of the stuff I have done with Tincr caches. Right now, I am simply focusing on making the import/export stuff stable, but I definitely have it on my TODO list!